astropy/wcs/tests/test_wcs.py

```diff
@@ -785,11 +786,17 @@ def test_validate_faulty_wcs():
 def test_error_message():
     header = get_pkg_data_contents("data/invalid_header.hdr", encoding="binary")
 
+    # make WCS transformation invalid
+    hdr = fits.Header.fromstring(header)
+    del hdr['PV?_*']
+    hdr['PV1_2'] = 110
+    hdr['PV2_2'] = 110
+
     with pytest.raises(wcs.InvalidTransformError):
         # Both lines are in here, because 0.4 calls .set within WCS.__init__,
```

Comment:
This is very old. Can we remove the `all_pix2world` line?

Reply:
It turns out that it is the pixel->world transformation that was causing the error that this test is supposed to catch. So we cannot get rid of `all_pix2world`.

Reply:
OK, fine then. And indeed good to remove the comment about versions that are long gone!